srcDLL/AppEvents.cpp

```diff
@@ -71,7 +51,10 @@ void TApp::ShutDown()
 	}
 }
 
-HINSTANCE WINAPI LoadShell(LPCSTR lpLibFileName)
+// Replacement method for call to LoadLibrary, which the game calls to load OP2Shell.dll
+// Must use WINAPI macro (__stdcall specifier) to ensure callee cleans the stack
+// By default, for plain functions, the caller cleans the stack, rather than the callee
```

Comment:
I know this is preexisting and just moved, but this is a good comment to have in the code.